.buildkite/nightly-benchmarks/scripts/run-performance-benchmarks.sh

```diff
@@ -69,6 +83,22 @@ json2args() {
   echo "$args"
 }
 
+json2envs() {
```

Comment:
I have a couple of questions on the method to run CPU benchmarks:

* Do we need to set CPU affinity to fix the number of core used by the benchmark?  I have seen similar step being done on PyTorch CPU benchmark, i.e. https://github.com/pytorch/pytorch/blob/main/.ci/pytorch/test.sh
* Do you know if we need to leave some CPU core + memory for Docker when running on CI?  A common issue I have seen is that the benchmark uses up all resources and the OS kills the runner daemon and fails the job